test/inductor/test_memory.py

```diff
@@ -0,0 +1,205 @@
+# Owner(s): ["module: inductor"]
```

Comment:
We might need to add this test file to this list as well:
https://github.com/pytorch/pytorch/blob/1dfa07e885dbd86f14030648d80517de384c6c6b/.ci/pytorch/test.sh#L1398-L1406

Reply:
also I wasn't able to find the new tests in the logs of a CI job yet... I spot-checked some of the jobs but couldn't find it :( we probably need to confirm it shows up in any job log to prove that it's running in CI

Reply:
the easiest way is probably to intentionally introduce an error in the unit test, and see if the CI fails as expected..

Reply:
Here is a screenshot showing that the purposely introduced error is indeed picked up by CI 

![image](https://github.com/user-attachments/assets/c6e3b152-451b-47ea-9dc9-868640875692)